interface/main/calendar/find_patient_popup.php

```diff
@@ -46,239 +46,254 @@
+                        echo "  <td class='srDOB'>" . text($iter['DOB']) . "</td>\n";
+                        echo "  <td class='srID'>" . text($iter['pubpid']) . "</td>\n";
+                        echo " </tr>";
+                    }
+                }
+                ?>
+                </tbody>
+            </table>
+
+        <?php endif; ?>
+
+        <script>
+
+            // jQuery stuff to make the page a little easier to use
+
+            $(function () {
+                $("#searchparm").focus();
+                $(".oneresult").mouseover(function () {
+                    $(this).toggleClass("highlight");
+                });
+                $(".oneresult").mouseout(function () {
+                    $(this).toggleClass("highlight");
+                });
+                $(".oneresult").click(function () {
+                    SelectPatient(this);
+                });
+                //ViSolve
+                $(".noresult").click(function () {
+                    SubmitForm(this);
+                });
+
+                //$(".event").dblclick(function() { EditEvent(this); });
+                $("#theform").submit(function () {
+                    SubmitForm(this);
+                });
+
+                $('select[name="searchby"]').on('change', function () {
+                    if ($(this).val() === 'DOB') {
+                        $('#searchparm').datetimepicker({
+                            <?php $datetimepicker_timepicker = false; ?>
+                            <?php $datetimepicker_showseconds = false; ?>
+                            <?php $datetimepicker_formatInput = true; ?>
+                            <?php require($GLOBALS['srcdir'] . '/js/xl/jquery-datetimepicker-2-5-4.js.php'); ?>
+                            <?php // can add any additional javascript settings to datetimepicker here; need to prepend first setting with a comma ?>
+                        });
+                    } else {
+                        $('#searchparm').datetimepicker("destroy");
+                    }
+                });
             });
-        } else {
-            $('#searchparm').datetimepicker("destroy");
-        }
-    });
-});
-
-function selpid(pid, lname, fname, dob) {
-    if (opener.closed || ! opener.setpatient)
-        alert("<?php echo htmlspecialchars(xl('The destination form was closed; I cannot act on your selection.'), ENT_QUOTES); ?>");
-    else
-        opener.setpatient(pid, lname, fname, dob);
-    dlgclose();
-    return false;
-}
 
-// show the 'searching...' status and submit the form
-var SubmitForm = function(eObj) {
-    $("#submitbtn").css("disabled", "true");
-    $("#searchspinner").css("visibility", "visible");
-    return true;
-}
+            function selpid(pid, lname, fname, dob) {
+                if (opener.closed || !opener.setpatient)
+                    alert("<?php echo text(xl('The destination form was closed; I cannot act on your selection.'), ENT_QUOTES); ?>");
```

Comment:
Have a left over ENT_QUOTES above
and it should be attr() or could go to xla() if drop the xl()
even better go to xlj (and drop outside quotes)